web/modules/custom/collation_fixer/collation_fixer.services.yml

```
@@ -0,0 +1,8 @@
+services:
+  collation_fixer.collation_fixer:
+    class: Drupal\collation_fixer\CollationFixer
+    arguments:
+      - "@database"
+      - "@module_handler"
+      - "@entity_type.manager"
+      - "@entity_field.manager"
```

```suggestion
      - "@entity_field.manager"
  Drupal\collation_fixer\CollationFixer: "@collation_fixer.collation_fixer"
```
Understøt autowiring?